flutter/shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -26,6 +26,10 @@ constexpr char kChannelName[] = "flutter/platform";
 constexpr char kGetClipboardDataMethod[] = "Clipboard.getData";
 constexpr char kSetClipboardDataMethod[] = "Clipboard.setData";
 constexpr char kClipboardHasStringsMethod[] = "Clipboard.hasStrings";
+
+constexpr char kIsLiveTextInputAvailable[] =
+    "LiveText.isLiveTextInputAvailable";
+
```

Comment:
It should make more sense if this line comes after `kSetSystemUIOverlayStyleMethod` (since `kIsLiveTextInputAvailable` comes after `kSetSystemUIOverlayStyleMethod` at line 166).

Reply:
done